tolstoy/src/syncer.rs

```diff
@@ -82,10 +100,14 @@ impl<'c> TxReceiver for UploadingTxReceiver<'c> {
         // TODO separate bits of network work should be combined into single 'future'
 
         // Upload all chunks.
-        for datom in d {
+        for datom in datoms {
             let datom_uuid = Uuid::new_v4();
             tx_chunks.push(datom_uuid);
-            self.remote_client.put_chunk(&datom_uuid, serde_cbor::to_vec(&datom)?)?
+            d(&format!("putting chunk: {:?}, {:?}", &datom_uuid, &datom));
+            // TODO switch over to CBOR once we're past debugging stuff.
```

Comment:
Interesting to note: Amazon Ion has a single flag to flip to switch between a CBOR-ish and a JSON-ish representation; no other code changes. Pretty neat.